examples/AlwaysMints.purs

```diff
@@ -28,6 +27,7 @@ import Contract.TxConstraints as Constraints
 import Contract.Value as Value
 import Contract.Wallet (mkNamiWalletAff)
 import Data.BigInt as BigInt
+import Contract.Scripts (MintingPolicy)
```

Comment:
I'm not sure why imports in unrelated modules have gotten changed so much on this branch